### ISSUE TITLE
SingleLinePerSelector: Bad SASS or linter bug?

### ISSUE THREAD

**Issue by chris-dura** [brigade/scss-lint#156]
So, I don't know if this is technically valid in SASS, if there's a better way to do it, please let me know, but it compiles fine:

``` scss
@mixin element-state() {
   $states: "";
   $states: $states + "&:active,";
   $states: $states + "&:focus,";
   $states: $states + "&:hover";

   #{$states} {
      color: red;
   }
}

// output
.widget:active,
.widget:focus,
.widget:hover {
   color: red;
}
```

But, I get this error with scss-lint:

``` bash
SCSSLint::Linter::SingleLinePerSelector raised unexpected error linting file src/sass/global/mixins/_element-state.scss: 'undefined method `gsub' for nil:NilClass'
```
